From 49979ce3c05f56da1dfa4b3ef281a852a07861a6 Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Fri, 7 Oct 2022 12:58:06 +0000
Subject: [PATCH] Declare TrebleApp permissions

Shouldn't be necessary but google do weird things every day. Let's avoid a possible future bootloop
---
 privapp-permissions-me.phh.treble.app.xml | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/privapp-permissions-me.phh.treble.app.xml b/privapp-permissions-me.phh.treble.app.xml
index 1aa71c3..2f51f94 100644
--- a/privapp-permissions-me.phh.treble.app.xml
+++ b/privapp-permissions-me.phh.treble.app.xml
@@ -1,8 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <permissions>
     <privapp-permissions package="me.phh.treble.app">
-        <permission name="android.permission.INTERACT_ACROSS_USERS"/>
+	    <permission name="android.permission.ACCESS_NETWORK_STATE"/>
+	    <permission name="android.permission.DIAGNOSTIC"/>
+	    <permission name="android.permission.INSTALL_PACKAGES"/>
+	    <permission name="android.permission.INTERACT_ACROSS_USERS"/>
+	    <permission name="android.permission.REQUEST_INSTALL_PACKAGES"/>
     </privapp-permissions>
+
     <!-- fix voice channels in calls (stop unloading treble-app) -->
     <allow-in-power-save package="me.phh.treble.app" />
 </permissions>
-- 
2.34.1

